lib/tasks/mailing_list.rake

```diff
@@ -1,32 +1,15 @@
 namespace :mailing_list do
   require 'services/mailing_list'
 
-  desc 'Subscribe all group members to relevant mailing list'
-  task update_all_subscribers: :environment do
-    groups = Group.all
+  desc 'Subscribe all active members to newsletter mailing list'
+  task subscribe_active_members: :environment do
+    newsletter_id = ENV['NEWSLETTER_ID'] || Rails.logger.info('NEWSLETTER_ID not set. Aborting task') && abort
 
-    groups.each do |group|
-      mailing_list = MailingList.new(group.mailing_list_id)
+    members = Member.includes(:subscriptions).where.not('subscriptions.member_id' => nil).uniq
+    newsletter = MailingList.new(newsletter_id)
 
-      group.members.each do |member|
-        puts mailing_list.subscribe(member.email, member.name, member.surname)
-      end
+    members.each do |member|
+      newsletter.subscribe(member.email, member.name, member.surname)
```

Comment:
Todo before merge: 
I missed checking and setting the `opt_in_newsletter_at` timestamp. 
This will also enable members to unsubscribe from the subscription page..